interviews.md

```diff
@@ -13,6 +13,7 @@ title: "Archive of King Gizzard interviews"
 * Oct 26: (video) [ABC News Australia](https://www.youtube.com/watch?v=rxle4toNKHU)
 * Oct 26: SPIN, [Supreme Ascendancy: Inside King Gizzard and the Lizard Wizard's New, Strange Trip](https://www.spin.com/2022/10/king-gizzard-and-the-lizard-wizard-three-albums-tour/) by Jonathon Cohen
 * Oct 11: [Stereogum - The Wild, Wonderful World Of King Gizzard & The Lizard Wizard](https://www.stereogum.com/2202315/king-gizzard-and-the-lizard-wizard-stu-mackenzie/interviews/cover-story/)
+* Oct 11: (video) Couch Radio Podcast, [Ep. 0016 Ambrose of The Murlocs and King Gizzard & The Lizard Wizard](Ep. 0016 Ambrose of The Murlocs and King Gizzard & The Lizard Wizard) - [(archived)](http://web.archive.org/web/20230324201132/https://www.youtube.com/watch?v=VCDIP8edmXQ)
```

Comment:
huh does YT work going through Archive?? the video doesn't play for me
<img width="1003" alt="Screenshot 2023-03-25 at 5 36 41 PM" src="https://user-images.githubusercontent.com/174307/227748971-264c16a9-00be-446f-b595-7fea3089575e.png">